kernel/ukvm/kernel.c

```diff
@@ -29,13 +29,16 @@ void _start(void *arg)
     platform_init(arg);
     cmdline = cmdline_parse(platform_cmdline());
 
+    console_init();
```

Comment:
I wonder if this call should be moved to before `cpu_init()`? That would in theory allow for console output if there's an early trap, assertion or other error in `cpu_init()` or `platform_init()`.

I haven't read through the muen channels and console init code in detail, would it work to call the initialisation this early? Ukvm should be fine as it's just a simple hypercall which will always work.